website_checker.py

```diff
@@ -116,6 +119,16 @@ def check_now(a,urls):
 			"name" : "WestBengal",
 			"url" : "https://www.wbhealth.gov.in/pages/corona/bulletin",
 			"check" : 'href="https://www.wbhealth.gov.in/uploaded_files/corona/'
+		},
```

Comment:
No `check` conditions for GJ and AP? We should add the corresponding frame that would change